### PR TITLE
Link to @vanpak

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,4 +22,4 @@ A .Net dynamic fake framework for creating all types of fake objects, mocks, stu
 * [Package](https://nuget.org/packages/FakeItEasy "FakeItEasy on NuGet")
 
 -
-FakeItEasy logo designed by Vanja Pakaski.
+FakeItEasy logo designed by [@vanpak](https://github.com/vanpak).
```
Comment (blairconrad) [inline README.md]: Not "by [Vanja Pakaski](https://github.com/vanpak)"?

I'm not fussy, but thought, "why not use her name?". Makes her sound more like a human.

Comment (thomaslevesque) [inline README.md]: agreed

Comment (adamralph) [inline README.md]: I'll ask here what she prefers :smile:

Comment (adamralph) [inline README.md]: I didn't get round to asking her, but I agree that keeping the name is better :smile: 

I've amended.

Comment (blairconrad) [inline README.md]: Fantastic. I was on the verge of tweeting her this morning. Then I saw the activity on her Twitter account and reconsidered.

Comment (adamralph) [inline README.md]: :laughing: no, that wouldn't have provoked much reaction.

**Pull request merged**